### DISCUSSION TITLE
Append Offset

### DISCUSSION THREAD

**Discussion by cryo75** [maxtoroq/DbExtensions#32] [Q&A] [answered]
Add the ability to append the offset using the following sql:

OFFSET {0} ROWS FETCH NEXT {1} ROWS ONLY

where {0} is the number of rows to skip and {1} is the number of results to return.

**Comment by maxtoroq** [comment, marked as answer]
You can do it already:

``` csharp
var query = SQL
   .SELECT("*")
   .FROM("Products")
   .ORDER_BY("ProductID")
   .OFFSET("{0} ROWS FETCH NEXT {1} ROWS ONLY", startIndex, pageSize);
```
